config/retired_zones/README

```diff
@@ -0,0 +1,2 @@
+Those zones have been retired for now to allow for a higher level overview of certain zones.
+They can be reused once an intermediate/state view has been implemented.
```

Comment:
Might want to add the .md extension to this file so it properly reads as a markdown file for all tools and systems.